vmngclient/dataclasses.py

```diff
@@ -3,14 +3,23 @@
 
 from attr import define, field
 
-from vmngclient.utils.creation_tools import FIELD_NAME, convert_attributes
+from vmngclient.utils.creation_tools import FIELD_NAME, asdict, convert_attributes
 from vmngclient.utils.device_model import DeviceModel
 from vmngclient.utils.personality import Personality
 from vmngclient.utils.reachability import Reachability
 
 
+class DataclassSettings:
+    def __str__(self):
+        return (
+            f'Class name: {self.__class__.__name__}'
+            + "\n"
+            + "\n".join(f'  {attribute[0]}: {attribute[1]}' for attribute in asdict(self).items())
```

Comment:
maybe make it look like in issue suggestion?
```python
Device(
    personality='vbond', 
    id='1.1.1.1'
)
```

```suggestion
            f"{self.__class__.__name__}(\n"
            + "\n".join(f"  {attribute[0]}: {attribute[1]}" for attribute in asdict(self).items())
            + "\n)"
```

Reply:
Have you tried
```python
json.dumps(self, indent=4)
```

Reply:
Couldn't dump enum to json